internal/connector/internal/handlers/connector_cluster.go

```diff
@@ -203,11 +203,17 @@ func (h *ConnectorClusterHandler) GetAddonParameters(w http.ResponseWriter, r *h
 		Action: func() (i interface{}, serviceError *errors.ServiceError) {
 
 			// To make sure the user can access the cluster....
-			cluster, err := h.Service.Get(r.Context(), connectorClusterId)
+			cluster, err := h.Service.Get(ctx, connectorClusterId)
 			if err != nil {
 				return nil, err
 			}
 
+			if r.URL.Query().Get("reset_credentials") == "true" {
```

Comment:
Just a suggestion: would be more true to the intent to call this something like `refresh_credentials` ?

Reply:
No, that's incorrect. It resets the credentials, the token will change. Hence the name.